Tests/configure_and_test_integration_instances.py

```diff
@@ -0,0 +1,563 @@
+    '''
+    Filter integrations into their respective lists - new, modified or unchanged. if it's on the skip list, then
+    skip if random tests were chosen then we may be configuring integrations that are neither new or modified.
+
+    Args:
+        integrations (list): The integrations to categorize.
+        skipped_integrations_conf (dict): Integrations that are on the skip list.
+        new_integrations_names (list): The names of new integrations.
+        modified_integrations_names (list): The names of modified integrations.
+
+    Returns:
+        (tuple): Lists of integrations objects as well as an Integration-to-Status dictionary useful for logs.
+    '''
+    new_integrations = []
+    modified_integrations = []
+    unchanged_integrations = []
+    integration_to_status = {}
+    for integration in integrations:
+        integration_name = integration.get('name', '')
+        if integration_name in skipped_integrations_conf.keys():
+            continue
+        elif integration_name in new_integrations_names:
+            new_integrations.append(integration)
+        elif integration_name in modified_integrations_names:
+            modified_integrations.append(integration)
+            integration_to_status[integration_name] = 'Modified Integration'
+        else:
+            unchanged_integrations.append(integration)
+            integration_to_status[integration_name] = 'Unchanged Integration'
+    return new_integrations, modified_integrations, unchanged_integrations, integration_to_status
+
+
+def get_integrations_for_test(test, skipped_integrations_conf):
+    '''Return a list of integration objects that are necessary for a test (excluding integrations on the skip list).
+
+    Args:
+        test (dict): Test dictionary from the conf.json file containing the playbookID, integrations and
+            instance names.
+        skipped_integrations_conf (dict): Skipped integrations dictionary with integration names as keys and
+            the skip reason as values.
+
+    Returns:
+        (list): List of integration objects to configure.
+    '''
+    integrations_conf = test.get('integrations', [])
+
+    if not isinstance(integrations_conf, list):
+        integrations_conf = [integrations_conf]
+
+    integrations = [
+        {'name': integration, 'params': {}} for
+        integration in integrations_conf if integration not in skipped_integrations_conf
+    ]
+    return integrations
+
+
+def update_content_on_demisto_instance(client, username, password, server):
+    '''Try to update the content
+
+    Args:
+        client (demisto_client): The configured client to use.
+        username (str): The username to pass to Tests/update_content_data.py
+        password (str): The password to pass to Tests/update_content_data.py
+        server (str): The server url to pass to Tests/update_content_data.py
+    '''
+    # TODO: need to add support for content packs
```

Comment:
what do you mean by that

Reply:
once we merge PR to use `demisto-sdk` in our circle config, the file path to the content zips will be different. so we will have to update this accordingly.

Reply:
updated.